x-pack/plugins/lens/public/drag_drop/providers.tsx

```diff
@@ -171,10 +171,10 @@ export function ReorderProvider({
       <EuiPortal>
         <EuiScreenReaderOnly>
           <div>
-            <p id="lnsDragDrop-reorderAnnouncement" aria-live="assertive" aria-atomic={true}>
+            <p aria-live="assertive" aria-atomic={true}>
               {state.keyboardReorderMessage}
             </p>
-            <p id={`lnsDragDrop-reorderInstructions-${id}`}>
```

Comment:
We can't merge this because this id is referenced from the buttons: https://github.com/elastic/kibana/blob/ac150da49def86136bfa3bb523ac28c37c90ae18/x-pack/plugins/lens/public/drag_drop/drag_drop.tsx#L389

We need it to tie the message and the button together

Reply:
Sorry, I thought I searched for it, but maybe I was wrong with the other one. Will reintroduce it back! 👍